code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -184,3 +184,31 @@
 			message = replacetextEx(message,regex(capitalize(key),"g"), "[capitalize(value)]")
 			message = replacetextEx(message,regex(key,"g"), "[value]")
 	speech_args[SPEECH_MESSAGE] = trim(message)
+
+/obj/item/clothing/mask/Madvilliansmask
```

Comment:
Change this entire type path to `/obj/item/clothing/mask/madvillain`, "mask" is implied by `/mask` and only snake_case names are allowed for type paths.